README.md

```diff
@@ -159,16 +159,27 @@ or alternatively set variable in `.npmrc`
 
 ## Download behind proxy
 
-In order to be able to download binaries when you're behind a proxy it will be enough to set http_proxy environment variable.
+In order to be able to download binaries when you're behind a proxy it will be enough to set the `http_proxy` environment
+variable. The `https_proxy` environment variable is supported as well, should the mirror be using the HTTPS protocol.
```

Comment:
isn't it the proxy server that's using HTTPS, and not the mirror?

Reply:
I guess both, right? It would be weird do proxy a HTTP mirror with HTTPS

Reply:
weird but not impossible

Reply:
ok, it's the download helper who will handle the proper proxy to use. it will all depend on the URL you download from. so I think it makes sense the way it's phrased here